x/twap/types/twap.pb.go

```diff
@@ -1,320 +0,0 @@
-// Code generated by protoc-gen-gogo. DO NOT EDIT.
-// source: osmosis/twap/v1beta1/twap.proto
-
-package types
```

Comment:
Note to reviewers: this is now in `genesis.go`. The proto file for this was missing